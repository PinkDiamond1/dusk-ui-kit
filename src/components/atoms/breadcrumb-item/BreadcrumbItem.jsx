/* @flow */
import * as React from 'react'
import BootstrapBreadcrumbItem from 'react-bootstrap/BreadcrumbItem'

type Props = {
  children: React.Node,
  href?: string,
}

const BreadcrumbItem = (props: Props): React.Element<*> => {
  const { children, href } = props
  // Extend defaults here...
  return <BootstrapBreadcrumbItem href={href}>{children}</BootstrapBreadcrumbItem>
}

BreadcrumbItem.defaultProps = {
  href: '#',
}

export default BreadcrumbItem
