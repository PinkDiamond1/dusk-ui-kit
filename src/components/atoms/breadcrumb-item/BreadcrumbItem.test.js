import React from 'react'
import renderer from 'react-test-renderer'

import BreadcrumbItem from './'

it('test to see if the BreadcrumbItem renders correctly', () => {
  const tree = renderer.create(<BreadcrumbItem>Smoke Test</BreadcrumbItem>).toJSON()
  expect(tree).toMatchSnapshot()
})
