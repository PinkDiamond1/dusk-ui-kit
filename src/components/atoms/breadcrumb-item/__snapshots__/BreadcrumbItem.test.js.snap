// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if the NavbarItem renders correctly 1`] = `
<li
  className="breadcrumb-item"
>
  <a
    href="#"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="button"
  >
    Smoke Test
  </a>
</li>
`;
